tests/devices.spec.ts

```diff
@@ -106,3 +106,44 @@ test("profile edit networks", async ({ page }) => {
 
   await deleteProfile(page, profile);
 });
+
+test("add, edit and remove an other device", async ({ page }) => {
+  const profile = randomProfileName();
+
+  //Add Other Device
+  await startProfileCreation(page, profile);
+  await page.getByText("Other", { exact: true }).click();
+  await page.getByRole("button", { name: "Attach custom device" }).click();
+  await page.getByLabel("Type").selectOption({ index: 6 });
+  await page.getByLabel("Bus Number").fill("123");
+  await page.getByLabel("Device Number").fill("12345");
+  await finishProfileCreation(page, profile);
+
+  //Edit and save Other Device
+  await visitProfile(page, profile);
+  await page.getByTestId("tab-link-Configuration").click();
+  await page.getByText("Other", { exact: true }).click();
```

Comment:
How about extracting a function for those 3 lines like `visitProfileOtherDevices(page, profile)`?